pkg/logs/client/http/destination.go

```diff
@@ -32,18 +34,32 @@ func NewDestination(endpoint config.Endpoint) *Destination {
 	}
 	return &Destination{
 		endpoint: endpoint,
-		url:      fmt.Sprintf("%v://%v/v1/input/%v", scheme, endpoint.Host, endpoint.UseSSL),
+		url:      fmt.Sprintf("%v://%v/v1/input/%v", scheme, endpoint.Host, endpoint.APIKey),
 		client: &http.Client{
 			Timeout:   time.Second * 10,
 			Transport: util.CreateHTTPTransport(),
 		},
+		destinationsContext: destinationsContext,
 	}
 }
 
 // Send sends a payload over HTTP.
 func (d *Destination) Send(payload []byte) error {
-	resp, err := d.client.Post(d.url, contentType, strings.NewReader(string(payload)))
```

Comment:
Why can't we stick to this method, it looks cleaner IMO ? We don't do anything fancy on the request.